plugins/catalog-graph/src/components/CatalogGraphCard/CatalogGraphCard.tsx

```diff
@@ -139,6 +141,8 @@ export const CatalogGraphCard = (
         mergeRelations={mergeRelations}
         direction={direction}
         relationPairs={relationPairs}
+        renderLabel={renderLabel}
```

Comment:
These should be collapsed into the `{...props}` above, we don't need to add them again here. The other options are defined here because there are default values provided either in lines 70-18 or in code before this block.

Reply:
@sennyeya  Updated. PTAL.